tests/integration/aof-multi-part.tcl

```diff
@@ -1110,18 +1110,14 @@ tags {"external:skip"} {
                 {file appendonly.aof.6.incr.aof seq 6 type i}
                 {file appendonly.aof.7.incr.aof seq 7 type i}
             }
-
-            set orig_size [r dbsize]
-            set load_handle0 [start_write_load $master_host $master_port 10]
-
-            wait_for_condition 50 100 {
-                [r dbsize] > $orig_size
-            } else {
-                fail "No write load detected."
+            
+            # Write more than 1KB data to trigger AOFRW
+            for {set j 0} {$j < 1024} {incr j} {
+                r set [expr rand()] [expr rand()]
```

Comment:
this isn't very efficient (many small keys which unknown size, and also no pipeline).
why not just a single `SETRANGE x 1024 1`?
or if we want many small keys `populate 1024`

Reply:
I think you misunderstood the mechanism of AOFRW's auto-triggering, redis compares the number of bytes written to the AOF file, not the number of keys in redis.
```
  if (server.aof_state == AOF_ON &&
            !hasActiveChildProcess() &&
            server.aof_rewrite_perc &&
            server.aof_current_size > server.aof_rewrite_min_size &&
            !aofRewriteLimited())
        {
             AOFRW();
        }
```

Reply:
ohh, right (not misunderstood, just forgot for a moment).
so scrap the SETRANGE suggestion.
it can be `SET x [string repeat x 1024]`, or `populate 1024` (to get multiple keys)

Reply:
`populate 1024` doesn't work，because it won't be written to AOF.
` r SET x [string repeat x 1024]` is a good suggestion.

p.s. I found that when changing `auto-aof-rewrite-min-size` to `1kb`, AOFRW will be triggered automatically (because a lot of data was written in the previous test, causing `server.aof_current_size` to exceed `1KB`). So at the beginning of the test, I first flush the data and executed an AOFRW to ensure that `server.aof_current_size` is 0.

Reply:
ok.. but not sure why `populate` won't work. it's a function (not DEBUG POPULATE).
```
# Write num keys with the given key prefix and value size (in bytes). If idx is
# given, it's the index (AKA level) used with the srv procedure and it specifies
# to which Redis instance to write the keys.
proc populate {num {prefix key:} {size 3} {idx 0}} {
    set rd [redis_deferring_client $idx]
    for {set j 0} {$j < $num} {incr j} {
        $rd set $prefix$j [string repeat A $size]
    }
    for {set j 0} {$j < $num} {incr j} {
        $rd read
    }
    $rd close
}
```

Reply:
ohh, I did understand it was `debug populate`, you are right.